Makefile

```diff
@@ -61,6 +67,7 @@ install:
 	install -D -m 755 $(CURDIR)/_output/nerdctl $(DESTDIR)$(BINDIR)/nerdctl
 	install -D -m 755 $(CURDIR)/extras/rootless/containerd-rootless.sh $(DESTDIR)$(BINDIR)/containerd-rootless.sh
 	install -D -m 755 $(CURDIR)/extras/rootless/containerd-rootless-setuptool.sh $(DESTDIR)$(BINDIR)/containerd-rootless-setuptool.sh
+	install -D -m 644 -t $(DESTDIR)$(DOCDIR)/nerdctl docs/*.md
```

Comment:
```suggestion
	install -D -m 644  $(CURDIR)/docs/*.md  $(DESTDIR)$(DOCDIR)/nerdctl
```

Reply:
Nice though, but doesn't work that way: the command won't create the "nerdctl" dir ('-D' creates only the leading path, but explicitly leaving off the last component). That's why I'm using '-t', which does create that dir.